packages/astro-parser/src/parse/read/expression.ts

```diff
@@ -84,7 +84,8 @@ function consume_tag(state: ParseState) {
 
     switch (char) {
       case "'":
-      case '"': {
+      case '"':
+      case '`': {
```

Comment:
This was the bug in the parser: we forgot to look for "`" as a valid string character.

This behavior was inconsistent with the rest of the parser which usually looks for all 3 string characters. And as far as I can tell, this deviation in this one spot seems accidental.

Reply:
Yep I think you're correct!

Reply:
This was me

Reply:
> This was me

I SAID **WE** MATTHEW THAT WASN’T AN ACCIDENT